### ISSUE TITLE
[luPopover] aria-expanded is not updated after closing

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#2857]
`aria-expanded` should be updated to `false` right after popover is closed.
It's currently updated after user hover component.
![chrome_zYHRp6A5E2](https://github.com/LuccaSA/lucca-front/assets/25581936/45ceafea-9511-4d4e-a903-132d0010ffe0)

**Comment by Supamiu** [comment]
I can't reproduce that in a storybook environment, might be specific to that use case?

I wrote tests in `fix/popover2-aria-expanded` branch

**Comment by jeremie-lucca** [comment]
@MarcFairbrother Is it still an issue on your side? 🤔

**Comment by MarcFairbrother** [comment]
@jeremie-lucca yes we still have the issue but we'll try to update the component to use popover2 instead
